lib/KeyboardShortcuts.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import React from 'react';
+import { Track } from 'livekit-client';
+import { useLocalParticipant, useTrackToggle } from '@livekit/components-react';
+
+export function KeyboardShortcuts() {
+  const _ = useLocalParticipant();
+  const { toggle: toggleMic, enabled: micEnabled } = useTrackToggle({ source: Track.Source.Microphone });
+  const { toggle: toggleCamera, enabled: cameraEnabled } = useTrackToggle({ source: Track.Source.Camera });
```

Comment:
This may be expected behavior, but I had to read `useLocalParticipant()` and discard the result in order to get the reactive behavior in `useTrackToggle`. Otherwise it never becomes aware of track source becoming available, or of the source changing. cc @lukasIO